src/modules/permalink/permalink-service.js

```diff
@@ -80,6 +110,14 @@ angular.module('anol.permalink')
                 result.groupOrder = groupOrder;
             }
 
+            if (angular.isDefined(sidebar)) {
+                result.sidebar = sidebar;
+            }
+
+            if (angular.isDefined(sidebarStatus) && sidebarStatus !== false && sidebarStatus !== '') {
```

Comment:
Is `sidebarStatus !== false` needed? Shouldn't it be either `'open'|'closed'`?

Reply:
yes that's true. I added some typechecking to avoid this kind of issues.